oozie-to-airflow/mappers/fs_mapper.py

```diff
@@ -42,59 +43,57 @@
 FS_TAG_GROUP = "group"
 
 
-def prepare_mkdir_command(node: Element, params):
-    path = normalize_path(node.attrib[FS_TAG_PATH], params)
-    command = "fs -mkdir -p {path}".format(path=shlex.quote(path))
-    return command
+def prepare_mkdir_command(node: Element, properties: Dict[str, str]) -> Tuple[str, List[str]]:
+    path = normalize_path_by_adding_hdfs_if_needed(node.attrib[FS_TAG_PATH], properties)
+    command = "fs -mkdir -p \\'{}\\'"
+    return command, [path]
```

Comment:
Maybe it's worth creating an type from this?

Reply:
Not now, really. I think it is quite OK for now - if i understand correctly that you are talking about  Tuple[str, List[str]] type. It's quite convenient to use it without class.

Reply:
Look at my idea presented in the form of a commit. https://github.com/GoogleCloudPlatform/cloud-composer/commit/63653f863614698c410e6b0177317b7cb9ec57da
If you have any questions, I will be happy to talk to you tomorrow at work.